### ISSUE TITLE
Add/Remove/Reorganize Settings and use new styling

### ISSUE THREAD

**Issue by rebron** [brave/brave-browser#955]
## Test plan
1. Open brave://settings
2. Check look and feel page 
    - are highlighted elements orange?
    - do the icons look like ours?
    - is the font Poppins?

## Description 
Add/remove/reorganize Brave settings and use new styling.

## Expected result
Make Brave settings more intuitive and consistent with Brave look and feel.

## Design
![df6efe0b-1fec-459f-9f83-dd8fb460a3d8 1](https://user-images.githubusercontent.com/5951041/46437183-97cc8500-c70f-11e8-9be8-23153dd8e9b9.png)

**Comment by rebron** [comment]
![core_settings_sample](https://user-images.githubusercontent.com/5951041/45057996-2b952d80-b04c-11e8-81ac-8d6d38cee049.png)

**Comment by rebron** [comment]
![settings-main](https://user-images.githubusercontent.com/5951041/45063312-35299000-b062-11e8-98d9-a913edcadb30.png)
most curret

**Comment by srirambv** [comment]
@bradleyrichter @rebron  `Reset Settings` has printer icon in the updated specs. should be updated

**Comment by rebron** [comment]
@srirambv Good catch, I'll get that updated. It'll be the same reset icon in the sidebar.

**Comment by beenotung** [comment]
I have got used to the current layout, wish this update doesn't come soon ...

**Comment by mil1i** [comment]
I would love to see the Appearance section have an option (and obviously the ability to) make the Open Tabs, Address Bar + Buttons and Bookmarks Bar different sizes. Preferably Compact.

As can be seen in the photo below, firefox I can have all 3 enabled and it cuts off where the Booksmarks Bar begins in Brave/Chrome.

![browser-bar](https://user-images.githubusercontent.com/8459426/47122656-ef511180-d234-11e8-8197-7f4e3aef49fb.png)

**Comment by Jacalz** [comment]
This looks really sweet, good job! 👍

**Comment by AlexeyBarabash** [comment]
Not sure, should it be here or be in a separate issue, but I see the overlapped controls on brave://bookmarks page

![image](https://user-images.githubusercontent.com/24739341/53964577-47ab8180-40f8-11e9-9a83-bfdee7e98a40.png)

Nightly master, Linux

Brave | 0.63.0 Chromium: 73.0.3683.39 (Developer Build) (64-bit)
-- | --
Revision | cc53b0e12fcaf42e4bab8d6c23bd4fb7aae99f6c-refs/branch-heads/3683@{#413}
OS | Linux

**Comment by srirambv** [comment]
@bsclifton @petemill once this reorganize/redesign of settings page is done I believe the side bar menu would be fixed? If so opening extensions page would hide the side bar and just show the existing extensions side bar options ?
![image](https://user-images.githubusercontent.com/17010094/55558183-83b31180-56b9-11e9-93be-f63c7a706c47.png)

User from https://community.brave.com/t/no-custom-keyboard-shortcuts-via-extensions-menu/51913 says the side bar menu on extensions page is missing so he is unable to access the keyboard shortcut option there.

**Comment by rebron** [comment]
@rossmoody mocks/spec to be updated to reflect decisions made by product and design during course of implementation.

**Comment by srirambv** [comment]
@petemill @rebron Couple of observations
- Only selected item icon is Orange and the text isn't bold. Is this expected?
- Should hover over settings sidebar have visual feedback

**Comment by srirambv** [comment]
Verification passed on 

Brave | 0.65.112 Chromium: 75.0.3770.38 (Official Build) beta(64-bit)
-- | --
Revision | 3860105745f2b12537da9e9f048f14c3f52ba970-refs/branch-heads/3770@{#618}
OS | Linux

* Verified based on discussion and test plan from https://github.com/brave/brave-core/pull/1909
* Confirmed everything works as expected with @petemill  regarding https://github.com/brave/brave-browser/issues/955#issuecomment-496288641
